### DISCUSSION TITLE
Transparency not rendered correctly in GIF?

### DISCUSSION THREAD

**Discussion by peteruithoven** [remotion-dev/remotion#3211] [Q&A] [answered]
Using the latest [v4.0.74](https://github.com/remotion-dev/remotion/releases/tag/v4.0.74) I'm trying to make a ping animation.
The following webm video gives an idea of what I'm going for: 
[Ping.webm](https://github.com/remotion-dev/remotion/assets/523210/c347c55f-c29a-4f61-8114-f451fcb1f685)

I'm planning to use this gif in Figma, so I can't just animate this just using some css.

But even though the background is transparent, the outer circle doesn't get transparent when I render it to a gif:
![Ping](https://github.com/remotion-dev/remotion/assets/523210/29112c99-893a-49d2-93af-76b172551766)

I've copied the project to StackBlitz, but sadly haven't figured out how to render from there:
https://stackblitz.com/edit/vitejs-vite-jokhgw?file=src%2FRoot.tsx

For rendering I've tried using:
- Codec: GIF
- Image Format: PNG
- Scale: 2x
- Pixel format: I've tried all except yuva420p
- Color space: tried both default and bt709 with a couple of pixel formats

**Comment by JonnyBurger** [comment, marked as answer]
We cannot support this because the GIF spec itself does not support partial transparency: https://stackoverflow.com/questions/1952376/how-to-get-better-transparency-with-gifs

**Reply by JonnyBurger** [reply]
Better + more references: https://github.com/ImageMagick/ImageMagick/discussions/6424 https://community.aseprite.org/t/i-need-to-make-a-gif-with-colors-that-become-transparent-or-semi-transparent-layers/7507

**Reply by peteruithoven** [reply]
Ahh :facepalm: Thanks for the answer!
